torch/onnx/symbolic.py

```diff
@@ -446,6 +446,15 @@ def leaky_relu(g, input, negative_slope, inplace=False):
     return g.op("LeakyRelu", input, alpha_f=_scalar(negative_slope))
 
 
+def linear(g, input, weight, bias=None):
+    if input.dim() == 2 and bias is not None:
+        return addmm(bias, input, weight)
```

Comment:
I don't think this matches the signature of addmm in this file, you need `g, self, mat1, mat2, beta, alpha`

Reply:
thanks